bundle/community/manifests/tempo-operator.clusterserviceversion.yaml

```diff
@@ -501,10 +503,22 @@ spec:
         - urn:alm:descriptor:io.kubernetes.conditions
       version: v1alpha1
   description: |
-    Tempo is an open source, easy-to-use, and high-scale distributed tracing backend, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki.
-    It can ingest common open source tracing protocols including Jaeger, Zipkin, and OpenTelemetry.
+    Tempo is an open source, easy-to-use, and high-scale distributed tracing backend.
+    It can ingest common open source tracing protocols including Jaeger, Zipkin, and OpenTelemetry and requires only object storage to operate.
+    Please refer to the [Tempo documentation](https://grafana.com/docs/tempo/latest/) for more information about Tempo.
 
     The Community Tempo Operator manages Tempo deployments in Microservices mode.
+
+    ### Operator features
```

Comment:
We should add the authn/authz to the list as well

Reply:
I've added:
**Authentication and Authorization**: Access to Tempo can be secured with RBAC

and I also copied the feature list to the readme.